frontend/packages/kubevirt-plugin/src/selectors/immutable/template/combined.ts

```diff
@@ -119,6 +119,14 @@ export const iGetCommonTemplateDiskBus = (tmp: ITemplate, diskName: string) => {
   return iGetIn(disk, ['disk', 'bus']);
 };
 
+export const iGetCommonTemplateDataVolumeSize = (tmp: ITemplate, diskName: string) => {
+  const dataVolume = iGetIn(iSelectVM(tmp), ['spec', 'dataVolumeTemplates'])?.find(
+    (d) => iGetIn(d, ['metadata', 'name']) === diskName,
+  );
+
+  return iGetIn(dataVolume, ['spec', 'pvc', 'resources', 'requests', 'storage']);
```

Comment:
`pvc` section was replace with `storage` section in dataVolume,
this should be something like:
`return iGetIn(dataVolume, ['spec', 'storage', 'resources', 'requests', 'storage']);`